Sprint 2 - UC00179_School_Enrolment_Trends

pooyafo opened the pull request:
In this sprint,
- Added more graphs to show enrolments across different years.
- Trained a model to predict future enrolments, which can help decision-makers plan and manage upcoming years more effectively.

MuzamilRafiq reviewed (approved):
Hi Pooya,

Great work on your use case. Below are my comments based on the pull request checklist:

Dataset Import: API and CSV datasets are used correctly, and no API key is exposed.

Code Quality & Readability: Code is well-structured with logical variable names.

Use Case Naming Conventions: Naming conventions have been followed correctly.

Pull Request Location: Submitted under the Playground folder.

Language: Markdown is clear and uses correct Australian English.

Visualisation: All charts and maps include proper legends, labels, and interpretations.

Tutorial Quality: Well formatted with step-by-step explanations and meaningful insights.

Use Case Template: Correct template has been followed.

Overall, this is a very strong and well-structured use case with valuable insights.

jennynguyen2112 reviewed (changes requested):
Hi Pooya,

Great work with your use case. Below are my comments based on the pull request checklist:
- Dataset Import: API is used correctly & API Key is not visible
- Code Quality & Readability: Code is well-structured. Variable names are logical and easy to follow.
- Use Case Naming Conventions: Use case naming conventions are followed with the code file.
- Pull Request Location: Pull request is correctly put in Playground folder
- Language: No spelling errors or incorrect use of language was found
- Visualisation: All visualisations have proper legends, labels and interpretations
- Tutorial quality: Use case is formatted with clear structure of a tutorial, containing explanation for each code chunk. However, it is lacking insights or conclusion drawn from each graph. Please add this to your use case and incorporate it into your conclusion section as well.
- Use Case Template: Correct template is used

Thank you Pooya!

pooyafo commented:
Hi @jennynguyen2112 
Thanks for your comments. I’ve added conclusion sections under each of the insight sections to summarise the findings more clearly. I’ve also updated the final conclusion at the end of the notebook to reflect the key takeaways from the analysis.
Thanks again!
